vars/neoDeploy.groovy

```diff
@@ -177,10 +193,15 @@ void call(parameters = [:]) {
 
                 StepAssertions.assertFileExists(this, configuration.source)
 
+                for(CharSequence extension in extensions) {
```

Comment:
change to `extensionList` or `extensionFile` for better distinction

Reply:
done